docs/yul.rst

```diff
@@ -1,18 +1,19 @@
-#################################################
-Joyfully Universal Language for (Inline) Assembly
-#################################################
+###
+Yul
+###
 
 .. _julia:
 
-.. index:: ! assembly, ! asm, ! evmasm, ! julia
+.. index:: ! assembly, ! asm, ! evmasm, ! yul, julia
```

Comment:
If we're keeping `julia` as search keyword, maybe keep `iulia` too? Otherwise I'd remove.